docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -4,89 +4,92 @@
 
 beta::[]
 
-{anomaly-detect-cap} alerts run scheduled checks on an {anomaly-job} or a group 
-of jobs to detect anomalies with certain conditions. If an anomaly meets the 
-conditions, the alert triggers the defined action. For example, you can create 
-an alert that checks an {anomaly-job} every fifteen minutes for critical 
-anomalies and notifies you in an email. This page helps you to configure an 
-{anomaly-detect} alert. To learn more about alerts in the {stack}, refer to 
+{anomaly-detect-cap} alerting runs scheduled checks on an {anomaly-job} or a 
+group of jobs to detect anomalies with certain conditions. If an anomaly meets 
+the conditions, an alert is created and the associated action is triggered. For 
+example, you can create a rule to check an {anomaly-job} every fifteen minutes 
+for critical anomalies and to notify you in an email. This page helps you to 
+configure {anomaly-detect} alerting. To learn more about alerting in the 
+{stack}, refer to 
 {kibana-ref}/alerting-getting-started.html#alerting-getting-started[Alerting and Actions].
 
 
-[[creating-anomaly-alerts]]
-== Creating an alert
+[[creating-anomaly-alert-rules]]
+== Creating a rule
 
-You can create {anomaly-detect} alerts in the {anomaly-job} wizard after you 
-start the job, from the job list, or under **{stack-manage-app} > {alerts-ui}**. 
-On the *Create alert* window, select *{anomaly-detect-cap} alert* under the 
-{ml-cap} section, then give a name to the alert and optionally provide tags.
+You can create {anomaly-detect} alerting rules in the {anomaly-job} wizard after 
```

Comment:
Not mandatory, but since they're just called "machine learning rules" in the Kibana docs, maybe that's sufficient here too:
```suggestion
You can create {ml} rules in the {anomaly-job} wizard after 
```